<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AltGui_Dam_Resettlement_Dia</class>
 <widget class="QDialog" name="AltGui_Dam_Resettlement_Dia">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>263</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set resettlement polygon...</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../system_sys/source_code/gui/resources/system_sys_resource.qrc">
    <normaloff>:/alt_icon</normaloff>:/alt_icon</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Select the file, which contains the polygon(s)-information where the resettlement should be located.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Sys_Filechooser_Dia" name="widget_file" native="true"/>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Select the biotop-type, which should be applied, after the resettlement to all element
in the polygon(s).</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="comboBox_btype">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>209</width>
         <height>18</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Sys_Filechooser_Dia</class>
   <extends>QWidget</extends>
   <header>sys_filechooser_dia.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../system_sys/source_code/gui/resources/system_sys_resource.qrc"/>
 </resources>
 <connections/>
</ui>
